cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -32,6 +32,7 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 
 	flags.Uint64("update-parallelism", 0, "task update parallelism (0 = all at once)")
 	flags.String("update-delay", "0s", "delay between task updates (0s = none)")
+	flags.String("update-failure-action", "pause", "action on failure during update (pause|continue)")
```

Comment:
`update-on-failure`?

As @cpuguy83 said, `OnFailure` is a good flag name but not a great proto name